GameOfRunes/GameOfRunes/GameEngine/Entities/PlayerUnitEntity.swift

```diff
@@ -13,13 +13,12 @@ class PlayerUnitEntity: Entity {
         .playerUnitEntity
     }
 
-    init(gameEngine: GameEngine?) {
+    init(sceneWidth: CGFloat) {
```

Comment:
I think it is okay to pass in gameEngine then retrieve the width. Seems weird to pass in sceneWidth when creating an Entity

Reply:
A bit of a nit-picking thing, but hmm if gameEngine is not needed, then maybe don't pass in gameEngine? Like we should only pass in what is absolutely necessary?

Reply:
erm im thinking cause it feels weird to pass in a width when creating entity hahaha. And perhaps pass in gameEngine is better in case we need more stuff in new components.